### PR TITLE
Feature Categorization using Satellite Imagery and Deep Learning

### PR DESCRIPTION
**Pull request by shivanip32**
\<insert pull request description here\>

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [x] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [x] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('https://www.arcgis.com', 'arcgis_python', 'P@ssword123')`
    - `gis = GIS(profile="your_online_profile")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal', 'arcgis_python', 'amazing_arcgis_123')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [ ] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [x] Code refactored & split out across multiple cells, useful comments?
- [x] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [x] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [x] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [x] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

**Comment by shivanip32** [comment]
https://github.com/ArcGIS/geosaurus/issues/3472

File: samples/04_gis_analysts_data_scientists/building_damage_assessment_using_feature_classifier.ipynb [hunk 1]
```diff
@@ -60,7 +60,12 @@
    "metadata": {},
```
Comment (rohitgeo) [inline samples/04_gis_analysts_data_scientists/building_damage_assessment_using_feature_classifier.ipynb:60]: <p><code>input_raster=input_raster_layer</code>  should work... do we need to use <code>input_raster_layer.url</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

**Comment by shivanip32** [comment]
Sure will update it.

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by AtmaMani** [comment]
@shivanip32 I have labeled this as 'do not merge'. Once you push your updates and ready to review, please remove it

**Comment by AtmaMani** [comment]
@rohitgeo are we targeting this PR for UC2020?

**Comment by rohitgeo** [comment]
@AtmaMani yes we'd like these updates to go in for the UC.

**Comment by AtmaMani** [comment]
@mohi9282 this sample has a name (and url) change

**Pull request merged**